webapp-client/src/App.css

```diff
@@ -0,0 +1,27 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 30px;
+  pointer-events: none;
```

Comment:
Just curious why this?

Reply:
this sets the size of the icon

Reply:
I meant pointer-events: none

Reply:
Ah I see! It came with the tutorial but I will remove it since when I removed it, nothing is affected.